docs/active_directory_ds.md

```diff
@@ -15,3 +20,72 @@ logging:
+
+## Configuration Options
+| Field               | Required | Default | Description                               |
+|---------------------|----------|---------|-------------------------------------------|
+| type                | required |         | Must be `active_directory_ds`.            |
+| collection_interval | optional | 60s     | A time.Duration value, such as 30s or 5m. |
+
+### Example Configuration
+
+```yaml
+metrics:
+  receivers:
+    active_directory_ds:
+      type: active_directory_ds
+  service:
+    pipelines:
+      active_directory_ds:
+        receivers:
+        - active_directory_ds
+```
+## Metrics
+
+The following metrics are collected from the Active Directory domain controller:
+| Metric                                                            | Data Type  | Unit              | Labels          | Description                                                                                                              |
+|-------------------------------------------------------------------|------------|-------------------|-----------------|--------------------------------------------------------------------------------------------------------------------------|
+| active_directory.ds.replication.network.io                        | CUMULATIVE | By                | direction, type | The amount of network data transmitted by the Directory Replication Agent.                                               |
+| active_directory.ds.replication.sync.object.pending               | GAUGE      | {objects}         |                 | The number of objects remaining until the full sync completes for the Directory Replication Agent.                       |
+| active_directory.ds.replication.sync.request.count                | CUMULATIVE | {requests}        | result          | The number of sync requests made by the Directory Replication Agent.                                                     |
+| active_directory.ds.replication.object.rate                       | GAUGE      | {objects}/s       | direction       | The number of objects transmitted by the Directory Replication Agent per second.                                         |
+| active_directory.ds.replication.property.rate                     | GAUGE      | {properties}/s    | direction       | The number of properties transmitted by the Directory Replication Agent per second.                                      |
+| active_directory.ds.replication.value.rate                        | GAUGE      | {values}/s        | direction       | The number of values transmitted by the Directory Replication Agent per second.                                          |
+| active_directory.ds.replication.operation.pending                 | GAUGE      | {operations}      | type            | The number of pending replication operations for the Directory Replication Agent.                                        |
```

Comment:
And this one doesn't -- I assume it just ended up on the wrong line and needs to be bumped up. That or the label in the metadata needs to be moved to this metric.